### ISSUE TITLE
sphinx_rtd_theme: Line numbers not aligned with code.

### ISSUE THREAD

**Issue by vbotka** [rtfd/readthedocs.org#3470]
## Details
Line numbers don't align with the code. Created with
```
.. highlight:: c                                                                            
    :linenothreshold: 5                                                                     
                                                                                            
.. literalinclude:: code/create-graph-01.c                                                  
    :language: c                                                                            
    :emphasize-lines: 37, 55, 74, 85, 89, 98, 107                                           
    :linenos:                                                                               
```

* Project URL:
http://redis-c-examples.readthedocs.io
http://redis-c-examples.readthedocs.io/en/latest/examples.html#create-frequency-graph-from-a-log

* Build URL (if applicable):
https://readthedocs.org/projects/redis-c-examples/builds/6487522/

* Read the Docs username (if applicable): 
vbotka

## Expected Result
Line numbers align with the code.

## Actual Result
Line numbers don't align with the code.

**Comment by vbotka** [comment]
Might be related to line-height.
 https://github.com/rtfd/sphinx_rtd_theme/commit/8355c2162fbb4bf0b216f15afdb9484bd791c649

I see local build aligned properly. When I compare the local build with RTFD I see larger vertical spacing between numbers at RTFD.

**Comment by vbotka** [comment]
http://redis-c-examples.readthedocs.io switched to guzzle_sphinx_theme.

**Comment by jessetan** [comment]
Looks like https://github.com/rtfd/sphinx_rtd_theme/issues/417, which comes from a bug in the theme code (related to line-height as suggested). This is fixed by https://github.com/rtfd/sphinx_rtd_theme/pull/504 which has landed on master and should be available in in upcoming (> 0.2.5b2) release of the theme.

**Comment by humitos** [comment]
@vbotka thanks for your report!

I'm closing this issue since it's not a RTD.org bug but a Sphinx Theme bug and it's tracked in the issues linked by @jessetan and the PR is already merged!

Feel free to reopen if you consider.
